provisioning/provisioning-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/provisioning/service/auth/ProvisioningServiceSasTokenTest.java

```diff
@@ -66,12 +66,12 @@ public void constructorSucceeded() throws Exception
             final SecretKeySpec secretKeySpec;
             Mac mac;
```

Comment:
what is Mac btw? :)

Reply:
https://docs.oracle.com/javase/7/docs/api/javax/crypto/Mac.html